### PR TITLE
feat(account page): update design

### PR DESCRIPTION
**Pull request by andreybakanovsky**

### CODE REVIEW

**Comment by andreybakanovsky** [comment]
image
image
image
image

**Comment by andreybakanovsky** [comment]
image
image
image

File: app/views/accounts/_objective.html.slim [hunk 1]
```diff
@@ -1,17 +1,50 @@
...
+      = objective.amount
   td
-    = link_to objective_path(objective), method: :delete, data: {confirm: 'Delete this objective?'}, class: 'button is-small is-light' do
-      span.icon
-        i.fa.fa-trash-o 
-      | &nbsp; Delete
+    .columns
+      .column.pb-0.is-pulled-left
+        p.is-black-budgeting-kid-color.pt-2
+          = "#{([objective.amount, 0].max / account.automatic_topup_configs.sum(:amount).+(0.1)).ceil} Weeks to go"
```
Comment (VladislavSokov) [inline app/views/accounts/_objective.html.slim:41]: Maybe we may move this into a separate method, like a objective.weeks_to_achieve method?

File: app/views/accounts/_transaction.html.slim [hunk 2]
```diff
@@ -1,30 +1,29 @@
...
+  td.is-vcentered.has-text-dark-gray-budgeting-kid-color
+    = transaction.description
+  - if transaction.from_account == account
+    td.is-vcentered= transaction.signed_amount(account)
+  - else
+    td.is-vcentered= transaction.signed_amount(account)
+  td.is-vcentered.px-0
+    = link_to transaction_path(transaction), method: :delete, data: {confirm: 'Cancel this transaction?'}, class: 'button is-white-budgeting-kid-color px-0' do
+      figure.image.is-24x24.is-round.is-budgeting-kid-light-color
+        = image_tag("cross-circle-white.svg")
```
Comment (VladislavSokov) [inline app/views/accounts/_transaction.html.slim:29]: Add empty line please

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -407,3 +425,15 @@ button.is-gray:hover {
...
+  background-color: hsla(243, 88%, 59%, 1) !important;
+}
+
+.progress.is-orange::-webkit-progress-value {
+  background-color: hsla(29, 88%, 59%, 1) !important;
+}
+
+.is-loupe-color {
+  filter: invert(15%);
+}
```
Comment (VladislavSokov) [inline app/assets/stylesheets/application.scss:439]: add empty line please

**Review by ka8725 — APPROVED** [review]
Thanks! It looks good.

**Pull request merged**